src/routes/Profile/index.js

```diff
@@ -12,17 +12,23 @@ import { authState } from '../../recoil/authRecoil';
 
 const Container = styled.div`
   width: 100%;
+  max-width:1280px;
```

Comment:
```suggestion
  max-width:1280px;
```
여기에 max를 주면 그 이상 너비일 때 이상하게 잘려 나옵니다
![스크린샷 2021-11-29 오후 3 47 31](https://user-images.githubusercontent.com/78777345/143821110-00c0ae88-419b-40fa-a3cb-aa19ded54b98.png)
이너 div를 하나 작성해서 거기에 주는 게 좋아보입니다.

Reply:
마진도 적용되서 배경색이 잘려 나오니 이 점도 반영해주세요